apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -61,7 +61,6 @@ paths:
                     
                     Example:
                        - `includeTaxNumbers=true`
-                  enum: ["true", "false"]
```

Comment:
I'm not clear on this... did this need to be removed?

Reply:
I ended up removing this to make it consistent with the others